earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
When you push commits up to the Bug-168_download-tutorial-file-script branch they are automatically added to the PR.